### DISCUSSION TITLE
Can I use this package on my Company's Application ?

### DISCUSSION THREAD

**Discussion by bhargavtenali** [motiondivision/motion#2184] [Q&A]
Hi,
This is not a technical doubt.
I want to know whether it is free to use in my company's project (based in react).
Or else should I purchase any subscription for using it in react project.
I need to animate a div component (showing and hiding).
I found framer-motion to be suiting my requirement.

Thanks
Bhargav

**Comment by MuhammadYousufHere** [comment]
Framer Motion is an open source, production-ready library that's designed for all creative developers.
So that means, your are allowed to use it.
